reference/6/Microsoft.PowerShell.Utility/Invoke-WebRequest.md

```diff
@@ -88,20 +88,22 @@ This command uses the `Invoke-WebRequest` cmdlet to send a web request to the Bi
 
 ```powershell
 $R = Invoke-WebRequest -URI http://www.bing.com?q=how+many+feet+in+a+mile
```

Comment:
This example does not work in v6 because we don't get a parsed HTML page. We only get a basic response. Also, have you tested the other examples in v6?